### PR TITLE
Create template for decision making RFC

### PR DESCRIPTION
**Pull request by snizhana-dynnyk**
towards https://github.com/giantswarm/giantswarm/issues/30631
this adds a template that can be copied to create an RFC for a technical decision. See https://github.com/giantswarm/rfc/pull/95

### CODE REVIEW

File: template/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+creation_date: 2024-05-29
```
Comment (JosephSalisbury) [inline template/README.md:2]: I'd remove all the specific details here (i.e: the date, etc.), so people can copy and paste and fill it out (or fill it with obviously junk versions)

File: template/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+### Who is affected / stakeholders
+<!-- List the individuals, teams, or SIGs that will be impacted by this decision and must provide feedback. -->
+
+### Preferred solution
+<!-- Describe the solution that is currently favored based on the analysis of the problem. -->
+
+### Alternative solutions
+<!-- Outline other potential solutions that were considered. For each alternative, provide a brief description and explain why it was not chosen as the preferred solution. -->
+
+### Implementation Plan
```
Comment (JosephSalisbury) [inline template/README.md:34]: The capitalisation here upsets my inner monk - is it specifically different here?

File: template/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+creation_date: 2024-05-29
+issues:
+- https://github.com/giantswarm/giantswarm/issues/30631
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+summary: This contains the RFC template for submitting a technical decision proposal
+---
+
+# This is a template that can be copied to create an RFC for submitting a technical decision proposal
```
Comment (JosephSalisbury) [inline template/README.md:11]: this should become the title?

File: template/README.md (hunk 2 above)
Comment (snizhana-dynnyk) [reply template/README.md:34]: no reason, just me becoming more German

Comment (snizhana-dynnyk) [inline template/README.md:34]:
```suggestion
### Implementation plan
```

Comment (JosephSalisbury) [reply template/README.md]: you've got to fight the change

Comment (JosephSalisbury) [reply template/README.md]: this isn't prufungsvorbereitung

**Review by JosephSalisbury — APPROVED** [review]

**Pull request merged**